README.md

```diff
@@ -28,7 +28,13 @@ AWS S3 bucket name to deploy to.
 
 #### `--cache-control <path>:<value> [<path>:<value>...]`
 
-Set CacheControl values for S3 path(s). Paths ending with a `*` will be treated as wildcards.
+Set CacheControl values for S3 path(s). Uses [micromatch](https://www.npmjs.com/package/micromatch) for file matching. More specific rules should come first. For example:
+```
+# will apply first rule to index.html and second rule to all other files
+--cache-control "/index.html:no-cache" "**:max-age=604800"
+# will apply first rule to index.html, second to all .svg files and last rule to all other files
+  --cache-control "/index.html:no-cache" "**/*.svg:max-age=31536000" "**:max-age=604800"
```

Comment:
Simplify example to:

    --cache-control "/index.html:no-cache" "**.svg:max-age=31536000" "**:max-age=604800"

Avoids mixing `**/*` and `**` syntax which behave the same but is confusing to see.

Reply:
Done